src/bobcat_miner/bobcat.py

```diff
@@ -202,17 +216,23 @@ def created(self):
         """Get miner created"""
         if not self.miner_data:
             self.refresh_miner()
-        return datetime.fromtimestamp(int(self.miner_data.get("miner", {}).get("Created")))
+        created = self.miner_data.get("miner", {}).get("Created")
+
+        if isinstance(created, int):
+            return datetime.fromtimestamp(int(created))
+        else:
+            return created
 
     @property
     def p2p_status(self):
         """Get p2p status"""
         if not self.miner_data:
             self.refresh_miner()
-        return {
-            x.split("|")[1].strip(): x.split("|")[2].strip()
-            for x in self.miner_data.get("p2p_status", [])[3:-3]
-        }
+        p2p_status = self.miner_data.get("p2p_status", [])
+        try:
+            return {x.split("|")[1].strip(): x.split("|")[2].strip() for x in p2p_status[3:-3]}
+        except Exception:
+            return "\n".join(self.miner_data.get("p2p_status", []))
```

Comment:
p2p_status will now return str when it fails to convert the bobcat data into a dict.